src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/HearingFeeUnpaidRespondentNotificationHandler.java

```diff
@@ -64,13 +67,16 @@ private CallbackResponse notifyRespondentSolicitorForHearingFeeUnpaid(
         CaseData caseData = callbackParams.getCaseData();
         String recipient = !is1v1Or2v1Case(caseData)
             && !isRespondent1(callbackParams, NOTIFY_RESPONDENT_SOLICITOR1_FOR_HEARING_FEE_UNPAID) ? caseData
-                .getRespondentSolicitor2EmailAddress() : caseData.getRespondentSolicitor1EmailAddress();
+                .getRespondentSolicitor2EmailAddress() : getRecipientRespondent1(caseData);
 
```

Comment:
Since respondent Lip email is not mandatory field and can be blank, we should add check and send email only in case if its available.